tracer/src/Datadog.Trace.ClrProfiler.Native/cor_profiler.cpp

```diff
@@ -523,6 +523,34 @@ bool ShouldRewriteProfilerMaps()
     return shared::TryParseBooleanEnvironmentValue(strValue, is_profiler_enabled) && is_profiler_enabled;
 }
 
+std::string GetNativeLoaderFilePath()
+{
+    auto native_loader_filename =
+#ifdef LINUX
+        "Datadog.Trace.ClrProfiler.Native.so";
+#elif MACOS
+        "Datadog.AutoInstrumentation.NativeLoader.dylib";
+#else
+        "Datadog.AutoInstrumentation.NativeLoader."
+#ifdef BIT64
+        "x64"
+#else
+        "x86"
+#endif
+        ".dll";
+#endif
+
+    auto module_file_path = fs::path(shared::GetCurrentModuleFileName());
+
+    auto native_loader_file_path =
+        module_file_path.parent_path() / ".." /
+#ifdef _WIN32
+        ".." / // On Windows, the tracer native library is 2 levels away from the native loader.
+#endif
```

Comment:
_sigh_ sure would be nice to unify all this lot 😄

Reply:
so true.